.github/workflows/cherryPick.yml

```diff
@@ -107,34 +107,43 @@ jobs:
         if: fromJSON(steps.cherryPick.outputs.HAS_CONFLICTS)
         id: createPullRequest
         run: |
-          gh pr create \
-            --title "🍒 Cherry pick PR #${{ github.event.inputs.PULL_REQUEST_NUMBER }} to staging 🍒" \
-            --body \
-              "🍒 Cherry pick https://github.com/Expensify/App/pull/${{ github.event.inputs.PULL_REQUEST_NUMBER }} to staging 🍒
-              This PR had conflicts when we tried to cherry-pick it to staging. You'll need to manually perform the cherry-pick, using the following steps:
+          # Download the author checklist
+          AUTHOR_CHECKLIST=$(curl -sL https://raw.githubusercontent.com/Expensify/App/refs/heads/main/.github/PULL_REQUEST_TEMPLATE.md | sed -n '/### PR Author Checklist/,$p')
```

Comment:
We have the repo checked out, do we still need to download this file?

Reply:
I reckon we could do some kind of `cat`

Reply:
updated